src/component/TodoFooter.tsx

```diff
@@ -0,0 +1,84 @@
+import classNames from 'classnames';
+import React, { useContext, useMemo } from 'react';
+import { TodosContext } from '../TodosContext';
+import { Status } from '../types/Status';
+
+export const TodoFooter:React.FC = () => {
+  const {
+    todos,
+    setTodos,
+    filterBy,
+    setFilterBy,
+    setIsToggleAllStatus,
+  } = useContext(TodosContext);
+
+  const uncompletedTodos = useMemo(() => {
+    return todos.filter(todo => !todo.completed).length;
+  }, [todos]);
+
+  const deleteTodos = () => {
+    setTodos(todos.filter(todo => todo.completed === false));
```

Comment:
Use logical operators
```suggestion
    setTodos(todos.filter(todo => !todo.completed));
```